LOMS/src/render/uiText.js

```diff
@@ -17,6 +17,15 @@ export default class UIText extends Actor {
             this._sprite.mousedown = (e) => {
                 this._onClick(e);
             }
+            this._sprite.mouseover = (e) => {
+                this._sprite.style = Object.assign({}, this._style, {
```

Comment:
Thank you for creating this PR, LOMS support ES6 , so maybe using spread operator? :)
You can find example [here](https://medium.com/@corinnemariekelly/object-assign-vs-spread-operator-577c889dbadc)